src/UBC Toboggan/Assets/Scripts/cameraFollow.cs

```diff
@@ -7,19 +7,20 @@ public class cameraFollow : MonoBehaviour
     public float followSpeed = 2f;
     public float yOffset = 1f;
     public float xOffset = 1f;
+    public float minX = 1f;
 
     public Transform target;
 
     void Start()
     {
-        Vector3 newPos = new Vector3(target.position.x + xOffset, target.position.y + yOffset,-10f);
+        Vector3 newPos = new Vector3(Mathf.Max(target.position.x + xOffset, minX), target.position.y + yOffset,-10f);
```

Comment:
Why did you change the new position formula?

Reply:
This is to snap the camera to the starting position, I changed it to add the ability to have a minimum X value the camera can go, this allows the player to slide into screen rather than starting with the player in the "middle of the screen"

Described here probably a little better: "Cleaned up level transition: At the start of the level the level designer can choose a minimum x position of the camera to allow the player to slide into view when the level is loaded."